shell/platform/linux/fl_dart_project.cc

```diff
@@ -82,12 +94,16 @@ static void fl_dart_project_class_init(FlDartProjectClass* klass) {
       G_OBJECT_CLASS(klass), PROP_ASSETS_PATH,
       g_param_spec_string(
           "assets-path", "assets-path", "Path to Flutter assets", nullptr,
-          static_cast<GParamFlags>(G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY |
-                                   G_PARAM_STATIC_STRINGS)));
+          static_cast<GParamFlags>(G_PARAM_READABLE | G_PARAM_STATIC_STRINGS)));
   g_object_class_install_property(
       G_OBJECT_CLASS(klass), PROP_ICU_DATA_PATH,
       g_param_spec_string(
           "icu-data-path", "icu-data-path", "Path to ICU data", nullptr,
+          static_cast<GParamFlags>(G_PARAM_READABLE | G_PARAM_STATIC_STRINGS)));
+  g_object_class_install_property(
```

Comment:
Nit: blank line above here (or removed above the other property) for consistency.